### PR TITLE
Add mpv2 to crowdstrike packs (not deprecated ones)

### PR DESCRIPTION
**Pull request by tcarmeli1**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-12222

## Description
Add mpv2 to crowdstrike packs (not deprecated ones):
  1. CrowdStrike Falcon Intel (Deprecated)
  2. CrowdStrike Falcon Sandbox (Deprecated)
  3. FalconHost (Deprecated)

### CODE REVIEW

File: Packs/CrowdStrikeFalcon/ReleaseNotes/2_0_27.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## CrowdStrike Falcon
+
+- Updated the pack to be on the *marketplacev2* marketplace.
```
Comment (ShirleyDenkberg) [inline Packs/CrowdStrikeFalcon/ReleaseNotes/2_0_27.md:3]:
```suggestion
Updated the pack to be on the *marketplacev2* marketplace.
```

File: Packs/CrowdStrikeFalconSandbox/README.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+CrowdStrike Falcon Sandbox is an automated malware analysis solution that empowers security teams by overlaying comprehensive threat intelligence with the results of the world's most powerful sandbox solution.
+
+This pack is uses a fully automated malware analysis using Hybrid Analysis API.
```
Comment (ShirleyDenkberg) [inline Packs/CrowdStrikeFalconSandbox/README.md:3]:
```suggestion
This pack is a fully automated malware analysis solution using the Hybrid Analysis API.
```

File: Packs/CrowdStrikeFalconSandbox/ReleaseNotes/2_0_23.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### CrowdStrike Falcon Sandbox v2 (Hybrid-Analysis)
+
+- Added the integration to the *marketplacev2* marketplace.
```
Comment (ShirleyDenkberg) [inline Packs/CrowdStrikeFalconSandbox/ReleaseNotes/2_0_23.md:6]:
```suggestion
Added the integration to the *marketplacev2* marketplace.
```

File: Packs/CrowdStrikeFalconStreamingV2/ReleaseNotes/1_1_14.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+## CrowdStrike Falcon Streaming
+
+- Updated the pack to be on the *marketplacev2* marketplace.
```
Comment (ShirleyDenkberg) [inline Packs/CrowdStrikeFalconStreamingV2/ReleaseNotes/1_1_14.md:3]:
```suggestion
Updated the pack to be on the *marketplacev2* marketplace.
```

File: Packs/CrowdStrikeFalconX/ReleaseNotes/1_2_40.md [hunk 5]
```diff
@@ -0,0 +1,3 @@
+## CrowdStrike Falcon Intelligence Sandbox
+
+- Updated the pack to be on the *marketplacev2* marketplace.
```
Comment (ShirleyDenkberg) [inline Packs/CrowdStrikeFalconX/ReleaseNotes/1_2_40.md:3]:
```suggestion
Updated the pack to be on the *marketplacev2* marketplace.
```

File: Packs/CrowdStrikeIntel/ReleaseNotes/2_0_39.md [hunk 6]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### CrowdStrike Falcon Intel v2
+
+- Added the integration to the *marketplacev2* marketplace.
```
Comment (ShirleyDenkberg) [inline Packs/CrowdStrikeIntel/ReleaseNotes/2_0_39.md:6]:
```suggestion
Added the integration to the *marketplacev2* marketplace.
```

File: Packs/CrowdStrikeMalquery/ReleaseNotes/1_0_29.md [hunk 7]
```diff
@@ -0,0 +1,3 @@
+## CrowdStrike Malquery
+
+- Updated the pack to be on the *marketplacev2* marketplace.
```
Comment (ShirleyDenkberg) [inline Packs/CrowdStrikeMalquery/ReleaseNotes/1_0_29.md:3]:
```suggestion
Updated the pack to be on the *marketplacev2* marketplace.
```

File: Packs/CrowdStrikeOpenAPI/ReleaseNotes/1_0_20.md [hunk 8]
```diff
@@ -0,0 +1,3 @@
+## CrowdStrike OpenAPI
+
+- Updated the pack to be on the *marketplacev2* marketplace.
```
Comment (ShirleyDenkberg) [inline Packs/CrowdStrikeOpenAPI/ReleaseNotes/1_0_20.md:3]:
```suggestion
Updated the pack to be on the *marketplacev2* marketplace.
```

**Comment by ShirleyDenkberg** [comment]
@jbabazadeh Doc review completed.

**Review by jbabazadeh — APPROVED** [review]
Great work!

**Comment by tcarmeli1** [comment]
Closing since the deprecation is being handled in https://jira-dc.paloaltonetworks.com/browse/CIAC-12439

**Pull request closed without merging**